metaschema-cli/src/main/java/gov/nist/secauto/metaschema/cli/commands/AbstractValidateContentCommand.java

```diff
@@ -194,7 +194,7 @@ public ExitStatus execute() {
           try {
             constraintSets.add(constraintLoader.load(constraintPath));
           } catch (IOException | MetaschemaException ex) {
-            return ExitCode.FAIL.exitMessage("Unable to load constraint set '" + arg + "'.").withThrowable(ex);
+            return ExitCode.INPUT_ERROR.exitMessage("Unable to load constraint set '" + arg + "'.").withThrowable(ex);
```

Comment:
Minor, non-blocking. If you like the above change:

```suggestion
            return ExitCode.IO_ERROR.exitMessage("Unable to load constraint set '" + arg + "'.").withThrowable(ex);
```